ana_scans.py

```diff
@@ -125,15 +125,22 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
   try:
     log = file("%s/anaLog.log"%(dirPath),"w")
  
-    #runCommand(cmd,log)
-    runCommand(cmd)
+    #returncode = runCommand(cmd,log)
+    returncode = runCommand(cmd)
```

Comment:
Why did the logging get removed?

Reply:
Wasn't me ;-) See bug #6

Reply:
OK, I guess I'll force @dorney to fix it in #23 since he introduced it originally

Reply:
Addressed in 311371684a8de283af669bf83ed0139eb4e76af5